pkg/core/resources/apis/meshservice/hostname/generator_test.go

```diff
@@ -24,12 +24,14 @@ import (
 
 var _ = Describe("MeshService Hostname Generator", func() {
 	var stopChSend chan<- struct{}
+	var store core_store.ResourceStore
```

Comment:
nit: do we need it?

Reply:
Ah no, I had this from when I tried with `memory.ClearStore`